src/edu/washington/escience/myria/operator/network/partition/PartitionFunction.java

```diff
@@ -23,6 +23,7 @@
     @Type(value = RoundRobinPartitionFunction.class, name = "RoundRobin"),
     @Type(value = SingleFieldHashPartitionFunction.class, name = "SingleFieldHash"),
     @Type(value = MultiFieldHashPartitionFunction.class, name = "MultiFieldHash"),
+    @Type(value = UnknownPartitionFunction.class, name = "Unknown"),
```

Comment:
I don't get it -- I need a subclass of PartitionFunction to indicate an unknown partition function. Are you saying I should just use a string "unknown" instead of a new class?